src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/jclouds/openstack/OpenstackJCloudsProvider.java

```diff
@@ -74,7 +81,7 @@
 
         NovaApi novaApi = computeService.getContext().unwrapApi((NovaApi.class));
 
-        ServerApi serverApi = novaApi.getServerApi(region);
+        ServerApi serverApi = novaApi.getServerApi(REGION);
 
         // Retrieve and add tags to the VM
         List<Tag> tags = tagManager.retrieveAllTags(instance.getOptions());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "tags" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1481)